setup.cfg

```diff
@@ -47,7 +47,7 @@ tests =
     mypy==0.961
     types-requests
 s3 =
-    moto[server]>=4.0.8
+    moto[server]>=4
```

Comment:
Changed back to `>=4` since `pytest-servers` can support all the packages in between. If `dql` needs latest, I think that should be required there.